annotation-api-1.2/pom.xml

```diff
@@ -38,7 +38,7 @@
         <dependency>
             <groupId>org.apache.servicemix.specs</groupId>
             <artifactId>org.apache.servicemix.specs.locator</artifactId>
-            <version>2.10-SNAPSHOT</version>
+            <version>2.10</version>
```

Comment:
Same point about the version here, maybe it would be better to use `1.2_1-SNAPSHOT`. Thoughts ?

Reply:
sure... please set it as you see fit...